docs/resources/resources-glossary.md

```diff
@@ -0,0 +1,132 @@
+
+See also: [ribbon, ribbon button](#ribbon-ribbon-button).
+
+## application
+
+In the Office Add-ins documentation, **application** refers to an Office application. The Office applications that support Office Add-ins are Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [client](#client), [host](#host), [Office application, Office client](#office-application-office-client).
+
+## application-specific API
+
+Application-specific APIs provide strongly-typed objects that interact with objects that are native to a specific Office application. For example, you call the Excel JavaScript APIs for access to worksheets, ranges, tables, charts, and more. Application-specific APIs are currently available for Excel, OneNote, PowerPoint, and Word. See [Application-specific API model](../develop/application-specific-api-model.md) to learn more.
+
+See also: [Common API](#common-api).
+
+## Content delivery network (CDN)
+
+A **content delivery network** or **CDN** is a distributed network of servers and data centers. It typically provides higher resource availability and performance when compared to a single server or data center.
+
+## client
+
+In the Office Add-ins documentation, **client** typically refers to an Office application. The Office applications, or clients, that support Office Add-ins are Excel, OneNote, Outlook, PowerPoint, Project, Visio, and Word.
+
+See also: [application](#application), [host](#host), [Office application, Office client](#office-application-office-client).
+
+## Common API
+
+Common APIs can be used to access features such as UI, dialogs, and client settings that are common across multiple Office applications. This API model uses [callbacks](https://developer.mozilla.org/docs/Glossary/Callback_function), which allow you to specify only one operation in each request sent to the Office application. Common APIs were introduced with Office 2013 and can be used to interact with Office 2013 or later. For details about the Common API object model, which includes APIs for interacting with Outlook, PowerPoint, and Project, see [Common JavaScript API object model](../develop/office-javascript-api-object-model.md).
+
+See also: [application-specific API](#application-specific-api).
+
+## Contoso
+
+**Contoso** Ltd. (also known as Contoso and Contoso University) is a fictional company used by Microsoft as an example company and domain.
+
+## content add-in
+
+**Content add-ins** are surfaces that can be embedded directly into Excel or PowerPoint documents. Content add-ins give users access to interface controls that run code to modify documents or display data from a data source. Use content add-ins when you want to embed functionality directly into the document. See [Content Office Add-ins](../design/content-add-ins.md) to learn more.
```

Comment:
I find the term "surfaces" mysterious. These are embedded webviews aren't they?

Reply:
I'm not familiar with the term "webview", and I don't see internet search results that consistently define this term. In term studio, I see the definition of webview is "A view that displays web content inside your application." Is this a term broadly understood by developers? I'm hesitant to use it, but I could use "webview" in this definition and then also define webview in the glossary. 

For reference, I pulled this definition: from https://docs.microsoft.com/en-us/office/dev/add-ins/design/content-add-ins

Reply:
> I'm not familiar with the term "webview", and I don't see internet search results that consistently define this term. In term studio, I see the definition of webview is "A view that displays web content inside your application." Is this a term broadly understood by developers? I'm hesitant to use it, but I could use "webview" in this definition and then also define webview in the glossary.
> 
> I agree that "surfaces" is ambiguous.
> 
> For reference, I pulled this definition: from https://docs.microsoft.com/en-us/office/dev/add-ins/design/content-add-ins

We use "webview" elsewhere in our docs. Sometimes we say "web control" or "web view control" or "embedded browser". People are familiar with it. The task pane is a webview. We should change that article to use "webview" too.